### ISSUE TITLE
Sign In page is not displayed in VSCode

### ISSUE THREAD

**Issue by MichaelSL** [alexa/ask-toolkit-for-vscode#119]
**Describe the bug**

When opening VSCode and clicking on `Sign In` button the page goes blank, browser page is not opened.

**To Reproduce**

1. Open VSCode
2. Click on Alexa Skills Toolkit Icon
3. Click on `Sign In` button
4. Pages goes blank, browser is not opened

**Extension Logs**

```
[info - 2021-04-07 03:25:38.354Z] Activating extension 
[debug - 2021-04-07 03:25:38.354Z] Registering commands in the extension 
[debug - 2021-04-07 03:25:38.355Z] Registering views in the extension 
[debug - 2021-04-07 03:25:38.414Z] Calling method: ask.login 
[debug - 2021-04-07 03:25:38.415Z] Calling method: initialLogin.getHtmlForView 
[debug - 2021-04-07 03:25:38.415Z] Calling method: renderView, args:  [{"name":"initialLogin","js":true}]
[debug - 2021-04-07 03:25:38.416Z] Registering event handlers in the extension 
[debug - 2021-04-07 03:25:38.416Z] Registering statusbar in the extension 
[verbose - 2021-04-07 03:25:38.416Z] Updating profile icon in the extension 
[debug - 2021-04-07 03:25:38.416Z] Registering treeviews in the extension 
[verbose - 2021-04-07 03:25:38.420Z] Calling method: findSkillFolders 
[verbose - 2021-04-07 03:25:38.774Z] Calling method: getOrInstantiateGitApi 
[verbose - 2021-04-07 03:25:38.774Z] Calling method: findSkillFolders 
[debug - 2021-04-07 03:25:38.775Z] Checking if extension version is latest 
[info - 2021-04-07 03:25:40.461Z] skill package schema has been updated. 
[verbose - 2021-04-07 03:25:40.461Z] Calling method: getSkillMetadataSrc 
[verbose - 2021-04-07 03:25:40.461Z] Calling method: getAskResourceConfig 
[info - 2021-04-07 03:25:40.462Z] SkillPackageWatcher is started 
```

**Expected behavior**

Browser window with Resource access grants should've been opened

**Screenshots**

![image](https://user-images.githubusercontent.com/2205113/113806068-46117500-9727-11eb-97f2-2bd757a1bac9.png)

**Desktop (please complete the following information):**

-   OS: Windows 10 21H1 OS Build 19043.906
-   Visual Studio Code Version: 1.56.0-insider
```
Visual Studio Code - Insiders
Version: 1.56.0-insider (system setup)
Commit: 550035808407c15252e581e8abe1b820b9d311ee
Date: 2021-04-02T05:15:44.248Z (4 days ago)
Electron: 11.4.1
Chrome: 87.0.4280.141
Node.js: 12.18.3
V8: 8.7.220.31-electron.0
OS: Windows_NT x64 10.0.19043
```
-   Alexa Skill Toolkit Version: v2.5.0

**Additional context**

I've found that WSL extension can interfere with ASK extension and disabled it. It didn't help.

**Comment by MichaelSL** [comment]
After switching to another page is VSCode and then returning to `Sign In` page it's rendered again:


![image](https://user-images.githubusercontent.com/2205113/113806687-5e35c400-9728-11eb-9d11-04aafbc2bdce.png)

**Comment by nikhilym** [comment]
Hey @MichaelSL . Sorry for responding late and thanks for getting this to our notice. I wasn't able to reproduce this issue on the latest VSCode version. Here's my system and extension information (retrieved from `About ASK Toolkit` command) : 

````
OS: Darwin x64 19.6.0
Visual Studio Code Version:  1.55.1
Alexa Skills Toolkit Version: 2.5.0
Git Version: git version 2.29.2

````

I'll try to reproduce this on the latest insiders version on a windows machine and get back to you. Glad that you are not blocked on this at the moment, and got it working.

**Comment by nikhilym** [comment]
Unable to reproduce this on the following insiders version, in a windows machine : 

````
OS: Windows_NT x64 10.0.17763
Visual Studio Code Version:  1.56.0-insider
Alexa Skills Toolkit Version: 2.5.0
Git Version: git version 2.27.0.windows.1

````
-----

> **Additional context**
> 
> I've found that WSL extension can interfere with ASK extension and disabled it. It didn't help.


Added the WSL extension and tested that the `sign in` page is rendered as expected.

**Comment by nikhilym** [comment]
Since we are unable to reproduce this issue at the moment, and since it is not blocking the user, I am closing this issue. @MichaelSL please reopen the issue in case you find it happening again or raise another issue if you have other questions around the extension. Thanks.

**Comment by MichaelSL** [comment]
@nikhilym it's happening not only on `Sign In` page. I've recorded a short video so you can see how it looks like on Simulator page.

https://user-images.githubusercontent.com/2205113/114500225-3abdbe00-9bed-11eb-8e2f-137bb03373f7.mp4
